enos/modules/docker_boundary/init.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Please use
```suggestion
#!/usr/bin/env bash
```
To make this file more portable